docs/README.md

```diff
@@ -0,0 +1,25 @@
+# .NET Performance Repository Documentation Readme
+
+The documentation in this repo is organized into the following sections:
+
+## Getting Started
+
+- [Prerequisites](prerequisites.md) - Information on what you need to get started.
+- [Perfreport walktrough](perfreport-walktrough.md) - This document describes Performance Report Advanced Features.
```

Comment:
nit: walktrough -> walkthrough this is also the case for the link itself

Perfreport -> Perf report

Reply:
thanks, fixed